security/rbac/scripts/submit_datagen_pageviews_config.sh

```diff
@@ -0,0 +1,31 @@
+#!/bin/bash
+
+CONNECT_HOST=localhost
+
+if [[ $1 ]];then
+    CONNECT_HOST=$1
+fi
+
+HEADER="Content-Type: application/json"
+DATA=$( cat << EOF
+{
+  "name": "datagen-pageviews",
+  "config": {
+    "connector.class": "io.confluent.kafka.connect.datagen.DatagenConnector",
+    "kafka.topic": "pageviews",
+    "quickstart": "pageviews",
+    "key.converter": "org.apache.kafka.connect.storage.StringConverter",
+    "value.converter": "org.apache.kafka.connect.storage.StringConverter",
+    "max.interval": 100,
+    "producer.override.sasl.mechanism": "OAUTHBEARER",
```

Comment:
Following up on comments left on the connect-avro-distributed.properties.delta file--we should remove the producer.override.sasl.mechanism and producer.override.sasl.login.callback.handler configs from this config and they should instead be set in the worker file.